Release 20240109

russbiggs opened the pull request.

dolugen commented:
Uh oh, I think I hijacked @mhamberg1's issue. I used `hub` to create the PR, and it replaced the original author as me. I don't think it's supposed to do that. That said, I shouldn't have used the "issue to PR" option in the first place.

mhamberg1 commented:
As a relatively n00b here I don't know what most of that means... But I'd be happy if you told me that the original request was solved.  :)

jflasher commented:
Thanks @dolugen, assuming my test passes, going to merge. It also took me way to long to actually wrap my head around this and what was going on for only 2 lines of code!

russbiggs committed:
add hint to 408 timeout error (#322)

russbiggs committed:
Fix/bad timeout value (#323)

---------

Co-authored-by: Christian Parker <c.andrew.parker@gmail.com>

* Bad timeout value passed to db class (#321)
  The averages endpoint was passing a config value but unassigned and
  it was therefor being passed as the timeout value. I added a check
  in the db to make sure we are protected from that as well as updated
  the averages endpoint to explicitly name the config argument.

fixes #321

---------

Co-authored-by: Christian Parker <c.andrew.parker@gmail.com>
